browser/src/map/Map.js

```diff
@@ -317,7 +312,7 @@ L.Map = L.Evented.extend({
 	},
 
 	initTextInput: function(docType) {
-		var hasAccessibilitySupport = window.enableAccessibility && this._accessibilityState;
+		var hasAccessibilitySupport = window.enableAccessibility;
```

Comment:
I'm not sure that this doesn't change behaviour outside of cypress... it looks to me like `_accessibilityState` is whether the user has toggled accessibility on and `enableAccessibility` is whether it's enabled in the server ... given that I didn't change that in my patch, I'm surprised you need to here

Was this intentional?